include/nbl/asset/utils/IIncludeHandler.h

```diff
@@ -16,7 +16,7 @@ namespace asset
 class IIncludeHandler : public core::IReferenceCounted
 {
 	public:
-		_NBL_STATIC_INLINE_CONSTEXPR char* BUILTIN_PREFIX = "nbl/builtin/";
+		static constexpr const char* BUILTIN_PREFIX = "nbl/builtin/";
```

Comment:
what about inline?

Reply:
you said yourself we dont need inline anymore?

Reply:
what, no. I said we dont need macro any more (macro was to not put `inline` on certain compilers)

Reply:
yea and in this case MSVC never got `inline`

Reply:
yep, it didnt support `inline`, now it's no problem (since we enabled cpp17)